src/components/lightbox/CaptionItem.tsx

```diff
@@ -0,0 +1,73 @@
+import { Button, Group, Stack, Textarea, Title } from "@mantine/core";
+
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import "react-virtualized/styles.css";
+import { Note } from "tabler-icons-react";
+import { useAppDispatch, useAppSelector } from "../../store/store";
+import { generatePhotoIm2txtCaption, editPhoto } from "../../actions/photosActions";
+
+type Props = {
+  isPublic: boolean;
+  photoDetail: any;
```

Comment:
You could try replacing the `any` here with the type `Photo` from (here)[https://github.com/LibrePhotos/librephotos-frontend/blob/dev/src/actions/photosActions.types.ts#L69]